Formula/osmfilter.rb

```diff
@@ -17,10 +17,11 @@ class Osmfilter < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  uses_from_macos "zlib"
```

Comment:
Please add a whitespace line.
```suggestion

  uses_from_macos "zlib"
```